core/services.go

```diff
@@ -276,30 +284,38 @@ func (app *Application) createEvent(clientID string, current *model.User, eventI
 		return nil, err
 	}
 
+	var userIDs []string
 	var recipients []notifications.Recipient
 	if len(event.ToMembersList) > 0 {
-		recipients = event.GetMembersAsNotificationRecipients(skipUserID)
+		userIDs = event.GetMembersAsUserIDs(skipUserID)
 	} else {
-		result, _ := app.storage.FindGroupMemberships(clientID, model.MembershipFilter{
-			GroupIDs: []string{group.ID},
-			Statuses: []string{"member", "admin"},
-		})
-		recipients = result.GetMembersAsNotificationRecipients(skipUserID)
+		userIDs = []string{group.ID}
+	}
+
+	result, _ := app.storage.FindGroupMemberships(clientID, model.MembershipFilter{
+		GroupIDs: userIDs,
```

Comment:
Why are we passing `userIDs` in as `GroupIDs`?

Reply:
Many thanks for the catch. It's a pure cognitive bias without an explanation :)